src/core/DataHelper.ttslua

```diff
@@ -923,6 +923,18 @@ PLAYER_CARD_TOKEN_OFFSETS = {
     { 0, 3, 0.5 },
     { -0.7, 3, 0.5 }
   },
+  [10] = {
+    { 0.7, 3, -0.9 },
+    { 0, 3, -0.9 },
+    { -0.7, 3, -0.9 },
+    { 0.7, 3, -0.2 },
+    { 0, 3, -0.2 },
+    { -0.7, 3, -0.2 },
+    { 0.7, 3, 0.5 },
+    { 0, 3, 0.5 },
+    { -0.7, 3, 0.5 },
+    { 0, 3, 1.2 }
+  },
   [12] = {
```

Comment:
Seems like we need 11 also

Reply:
There is no possible way to get 11 at the moment

Reply:
Worth adding though so it's covered and doesn't get missed if something else happens to hit that mark.